Backend/dev/migrations/rider-app/1199-adding-enabled-to-ticket-place-and-service.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE atlas_app.ticket_place ADD COLUMN enabled boolean ;
+ALTER TABLE atlas_app.ticket_service ADD COLUMN enabled boolean ;
+
+UPDATE atlas_app.ticket_place SET enabled = true;
+UPDATE atlas_app.ticket_service SET enabled = true;
```

Comment:
This also looks manually written. Also, why is this field optional? Can we make this mandatory and use `default` in yaml to set the values?